scripts/validate_erlang_config.erl

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env escript
+main([ConfigFile]) ->
+    {ok, Terms} = file:consult(ConfigFile),
+    io:format("~p~n",[Terms]).
```

Comment:
Is this for manual testing purposes, or does something execute this that I'm not seeing?